topics/sentinel.md

```diff
@@ -801,7 +798,18 @@ When configured this way, Sentinels will do two things:
 
 This means that **you will have to configure the same `requirepass` password in all the Sentinel instances**. This way every Sentinel can talk with every other Sentinel without any need to configure for each Sentinel the password to access all the other Sentinels, that would be very impractical.
 
-Before using this configuration make sure your client library is able to send the `AUTH` command to Sentinel instances.
+Before using this configuration, make sure your client library can send the `AUTH` command to Sentinel instances.
+
+With Redis 6.2, Sentinel instances can be configured with ACL authentication, so clients need to provide both a user and a password. As with password-only authentication, the same ACL user is used to authenticate both incoming clients and outgoing Sentinel connections. You can create and manage the Sentinel ACL user on all instances with regular [ACL directives](/topics/acl), for example:
+
+    ACL SETUSER my-sentinel-user on >my-sentinel-password +@all 
```

Comment:
@hwware is this a good example? Perhaps something more restrictive?

Reply:
@itamarhaber oh yes, maybe  give just enough permission for the sentinel user, let me test this.. thanks for catching up this.